### PR TITLE
[chore] Update meshkit and docker

### PR DESCRIPTION
**Pull request by nebula-aac**
**Description**

This PR updates `meshkit` to `v0.6.48` and updates docker to `v20` instead of `v17`.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@aritra8438 Do you mind reviewing this PR?

**Comment by leecalcote** [comment]
Conflict resolved.

**Comment by Aritra8438** [comment]
> @Aritra8438 Do you mind reviewing this PR?

Will try my best @leecalcote

**Comment by leecalcote** [comment]
I can't assign your user to a review, but you should be able to offer comment when you do review, @Aritra8438

File: go.mod [hunk 1]
```diff
@@ -11,6 +10,7 @@ go 1.19
 require (
 	github.com/buger/jsonparser v1.1.1
 	github.com/google/uuid v1.3.0
+	github.com/layer5io/meshkit v0.6.48
 	github.com/layer5io/meshkit v0.6.50
```
Comment (Aritra8438) [inline go.mod:14]: Why are we importing different versions of same package?

As the package is maintained by us only, can we not make changes to meshkit to fix this issue?

Comment (Aritra8438) [reply go.mod:14]: And, why importing an older version?

File: go.mod [hunk 2]
```diff
@@ -11,6 +10,7 @@ go 1.19
 require (
 	github.com/buger/jsonparser v1.1.1
 	github.com/google/uuid v1.3.0
+	github.com/layer5io/meshkit v0.6.48
```
Comment (leecalcote) [inline go.mod:13]:
```suggestion
```

**Comment by leecalcote** [comment]
@nebula-aac is this ready for merge?

**Comment by leecalcote** [comment]
@Aritra8438 how are we looking now?

**Pull request merged**